Ginger/GingerCore/Drivers/SeleniumDriver.cs

```diff
@@ -502,8 +557,12 @@ public override void StartDriver()
 
                             break;
                         }
+                        #endregion
                 }
 
+                if (BrowserMinimized == true && mBrowserTpe != eBrowserType.Edge)
```

Comment:
There's an Exception here as, "Exception in start driver OpenQA.Selenium.WebDriverException: Unexpected error. Unknown command received". Thus added a check to ignore calling Window.Minimze() for Edge Browser.